assets/tests/__mocks__/axios.js

```diff
@@ -1,6 +1,18 @@
 const fs = require('fs')
 
-const mockIndex = jest.fn((uri) => {
+const mockGet = jest.fn((uri) => {
+  setLastURI(uri);
+  if (uri === "/api/goals/achieved") {
+    return new Promise((resolve, reject) => {
+      fs.readFile(`./tests/__mockData__${uri}.json`, (err, data) => {
+        if (err) {
+          reject(err)
+        }
+        resolve(JSON.parse(data))
```

Comment:
It would be great if we could inject this data so it could be different based on the mocking scenario.

Reply:
+1